gensim/models/wrappers/fasttext.py

```diff
@@ -251,6 +251,14 @@ def load_fasttext_format(cls, model_file, encoding='utf8'):
         model.load_binary_data(encoding=encoding)
         return model
 
+    @classmethod
+    def load(cls, *args, **kwargs):
+        model = super(FastText, cls).load(*args, **kwargs)
+        if hasattr(model.wv, 'syn0_all'):
+            setattr(model.wv, 'syn0_ngrams', model.wv.syn0_all)
```

Comment:
What's with this `setattr` / `delattr`? Why not a simple (and more readable) `model.wv.syn0_ngrams = model.wv.syn0_all`?